internal/metamorph/server.go

```diff
@@ -136,7 +136,6 @@ func (s *Server) StartGRPCServer(address string, grpcMessageSize int, prometheus
 
 func (s *Server) Shutdown() {
 	s.logger.Info("Shutting down")
-	s.processor.Shutdown()
```

Comment:
Don't we need this?

Reply:
I think not, I found out we are calling `processor.Shutdown()` twice and caused race conditions in tests.

Reply:
I believe we don't call it twice now. I have changed it in #429